server/src/test/java/com/broadcom/lsp/cobol/ConfigurableTest.java

```diff
@@ -14,20 +14,29 @@
  */
 package com.broadcom.lsp.cobol;
 
-import com.broadcom.lsp.cobol.domain.modules.EngineModule;
-import com.broadcom.lsp.cobol.domain.modules.LangServerCtx;
 import com.broadcom.lsp.cobol.domain.modules.DatabusModule;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.BeforeAll;
+import com.broadcom.lsp.cobol.domain.modules.EngineModule;
+import com.broadcom.lsp.cobol.positive.CobolTextRegistry;
+import com.broadcom.lsp.cobol.positive.FolderTextRegistry;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import org.junit.jupiter.api.TestInstance;
 
+import static java.lang.System.getProperty;
+import static java.util.Optional.ofNullable;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public abstract class ConfigurableTest {
-  @BeforeAll
-  public static void setUp() {
-    LangServerCtx.getGuiceCtx(new TestModule(), new EngineModule(), new DatabusModule());
-  }
+  protected static final String PATH_TO_TEST_RESOURCES = "filesToTestPath";
+  protected Injector injector =
```

Comment:
Why is it protected?

Reply:
Yes, correct need not be. Private would suffice.

Reply:
resolved on chat.